roles/edpm_bootstrap/defaults/main.yml

```diff
@@ -77,3 +77,5 @@ edpm_bootstrap_swap_partition_label: swap1
 # Shell command that is executed before any packages are installed by the role.
 # Can be used to register systems using any arbitrary registration command(s).
 edpm_bootstrap_command: ""
+
+edpm_bootstrap_command_done_file: /.bootstrap_command_done
```

Comment:
I know it's in arg spec, but it still ought to have a comment.

Reply:
:+1: